java-bootcamp-2018-tdd/src/main/java/com/globant/business/shoppingCart/Product.java

```diff
@@ -0,0 +1,91 @@
+package com.globant.business.shoppingCart;
+
+public class Product {
+	/**
+	 * simple product, Name, Description and price
+	 */
+	private String name;
+	private String category;
+	private double price;

```

Comment:
A double is huge, with a float you can sell really expensive stuff and save half the memory ;)